### ISSUE TITLE
Update accessibility statement when iterated accordion goes live

### ISSUE THREAD

**Issue by EoinShaughnessy** [alphagov/govuk-design-system#1924]
## What

Update [our accessibility statement](https://design-system.service.gov.uk/accessibility/) to remove mention of the [current WCAG fail](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) for the [accordion component](https://design-system.service.gov.uk/components/accordion/).

## Why

The iterated component will fix the accessibility issue, where section headings can be mistaken for links, but are treated as buttons.

## Who needs to know about this

Technical Writer

## Done when

- [ ] Technical Writer updates accessibility statement

**Comment by EoinShaughnessy** [comment]
This duplicates [#1823](https://github.com/alphagov/govuk-design-system/issues/1823), so I'm closing it.
